daemon/mgr/container.go

```diff
@@ -322,9 +321,9 @@ func (mgr *ContainerManager) Create(ctx context.Context, name string, config *ty
 		return nil, errors.Wrapf(errtypes.ErrInvalidParam, "unknown runtime %s", config.HostConfig.Runtime)
 	}
 
-	config.Image = primaryRef.String()
+	snapID := id
```

Comment:
Could you explain why to use generated container ID to assign to snapshot ID?

Reply:
SnapshotID is used to equal with ContainerID, but after `upgrade` they are different.

So, i use param `snapID` here just to reminder that snapshot id and container id are different